core/src/components/title/title.ios.scss

```diff
@@ -14,7 +14,7 @@
 
   transform: translateZ(0);
 
-  font-size: 17px;
+  font-size: dynamic-font-max(17px, 1.2);
```

Comment:
![Screenshot 2023-08-29 at 4 49 54 PM](https://github.com/ionic-team/ionic-framework/assets/13530427/d7637c8a-2fb0-468c-8c89-2ecc4a7aa366)

I used the Contacts app as a reference